lcs/agents/xcs/Classifier.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from typing import Union, Optional
+
+from lcs.agents.xcs import Configuration, Condition
+
+logger = logging.getLogger(__name__)
+
+
+class Classifier:
+    def __init__(self,
+                 cfg: Optional[Configuration] = None,
+                 condition: Union[Condition, str, None] = None,
+                 action: Optional[int] = None,
+                 time_stamp: int = None) -> None:
+        if cfg is None:
+            raise TypeError("Configuration should be passed to Classifier")
+        if type(condition) != Condition:
+            condition = str(condition)
+        self.cfg = cfg                  # cfg
+        self.condition = condition      # current situation
+        self.action = action            # A - int action
+        self.time_stamp = time_stamp    # time - int
+        self.experience = 0             # exp
+        self.action_set_size = 1        # as
```

Comment:
Czy `action_set_size` jest cechą klasyfikatora?

Reply:
Jest cechą klasyfikatora. Przy okazji spróbuję wgrać zdjęcie z Algorithmic Description of XCS gdzie o tym mówią. 
![image](https://user-images.githubusercontent.com/27001470/111978385-1f86f380-8b04-11eb-82b0-dfd7d1572bcd.png)